cloudpickle/cloudpickle.py

```diff
@@ -269,7 +269,7 @@ class CloudPickler(Pickler):
 
     dispatch = Pickler.dispatch.copy()
 
-    def __init__(self, file, protocol=None):
+    def __init__(self, file, protocol=None, override_existing_globals=True):
```

Comment:
In retrospect I find `override_existing_globals` a bit too verbose and redundant. May I suggest to rename this to `override_globals`.